src/app/colors/colors.model.ts

```diff
@@ -1 +1,28 @@
 export type ColorAndTooltip = readonly { color: string; tooltip: string }[];
+
+export const typeColors: ColorAndTooltip = Object.freeze([
+    {
```

Comment:
mhm, in this way you cannot reuse the `ColorAndTooltip` type, because it's an array, moreover if you are declaring a type "array" you should use a plural name like "ColorsAndTooltips" or something that make you understand that is an array.

I would do it diffferently
```typescript
export type ColorAndTooltip = readonly { color: string; tooltip: string };

export const typeColors: ColorAndTooltip[] = Object.freeze([
```

So I moved the `[]` from the type to the typeColors declaration to re-use possibly the ColorAndTooltip type.

Reply:
I intended to do the same actually, but instead I made a mixture 😆